include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -347,10 +431,9 @@ class alignment_algorithm :
         {
             this->next_column(); // Move to the next column.
             auto col = this->current_column();
+            this->init_row_cell(*std::ranges::begin(col), cache); // initialise first row of dp matrix.
 
-            this->init_row_cell(*seqan3::begin(col), cache); // initialise first row of dp matrix.
-
-            auto second_range_it = seqan3::begin(second_range);
+            auto second_range_it = std::ranges::begin(second_range);
```

Comment:
I think these changes are not necessary since we will replace all these lines anyway?

Reply:
IDC. I regexed. Why would I change them back?